src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -412,3 +430,28 @@
     {%- endif -%}
   {%- endfor -%}
 {%- endblock attributes -%}
+
+{%- block addons_connected_information -%}
```

Comment:
Same here: if we dont reuse it, please do not make widgets as I'm worried about premature abstraction